js/web/lib/wasm/jsep/webgpu/gpu-data-manager.ts

```diff
@@ -460,6 +468,11 @@ class GpuDataManagerImpl implements GpuDataManager {
     this.capturedPendingBuffers = new Map();
   }
 
+  onCreateSession() {
+    LOG_DEBUG('verbose', () => '[WebGPU] Incrementing session count');
```

Comment:
don't think we need logging here since there are already plenty of logs indicating new sessions